<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>612</width>
    <height>380</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>612</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidget">
     <property name="minimumSize">
      <size>
       <width>5</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>180</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
     </property>
     <property name="iconSize">
      <size>
       <width>48</width>
       <height>48</height>
      </size>
     </property>
     <property name="currentRow">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>General</string>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/assets/general.png</normaloff>:/assets/general.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Video</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/assets/video.png</normaloff>:/assets/video.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Audio</string>
      </property>
      <property name="font">
       <font>
        <stylestrategy>PreferDefault</stylestrategy>
       </font>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/assets/music.png</normaloff>:/assets/music.png</iconset>
      </property>
      <property name="flags">
       <set>ItemIsSelectable|ItemIsEnabled</set>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="General">
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>321</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>No Settings Currently Available In This Section.</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Video">
      <widget class="QCheckBox" name="checkBox_enable_highres">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable High Resolution</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_force_bilinear_filtering">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Force Bilinear Filtering</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_presentation_mode">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>161</width>
         <height>27</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Fill Screen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fit Screen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Original Size</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>151</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Presentation Mode:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="Audio">
      <widget class="QCheckBox" name="checkBox_enable_audio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>341</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Enable Audio Output</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
